src/lib/storage/fake_device_factory.rb

```diff
@@ -212,9 +212,9 @@ def set_topology_attributes!(disk, args)
         io_size = args["io_size"]
         min_grain = args["min_grain"]
         align_ofs = args["align_ofs"]
-        disk.topology.optimal_io_size = io_size.size if io_size && io_size.size > 0
+        disk.topology.optimal_io_size = io_size.size if io_size && !io_size.empty?
```

Comment:
io_size is the optimal input/output size of the disk. So it really represents a size. Technically it is a ```DiskSize``` object. The ```DiskSize``` object has a ```zero?``` method but not an ```empty?``` method (which is correct).

AFAIS the change breaks the testsuite:

> undefined method `empty?' for <DiskSize 2.00 GiB (2147483648)>:Yast::Storage::DiskSize